### DISCUSSION TITLE
Sentry

### DISCUSSION THREAD

**Discussion by blasdfaa** [coollabsio/coolify#2993] [Service Template Requests]
https://sentry.io/ - Sentry provides end-to-end distributed tracing, enabling developers to identify and debug performance issues and errors across their systems and services.

**Comment by Iulian-Dragomirescu** [comment]
this is a **good** idea!

**Comment by JailbreakPapa** [comment]
I would also want to see this in coolify!

**Comment by tconroy** [comment]
Just chiming in to +1 the above -- Sentry is a must-have for JS apps!

**Comment by fedy97** [comment]
+1

**Comment by MaxLikesCode** [comment]
+1

**Comment by mksglu** [comment]
+1

**Comment by Bilge** [comment]
Must-have for PHP also.

**Comment by pirmax** [comment]
+1

**Comment by Karnak19** [comment]
+1

**Comment by BohdanK-W32** [comment]
+1

**Comment by developerautomationkit** [comment]
+1

**Comment by Bilge** [comment]
Can you clowns ever stop posting +1s? Every time you send an emails to hundreds of people. Click the upvote button. That's what it's there for.

**Reply by johan-t** [reply]
+1

**Reply by BohdanK-W32** [reply]
Feel free to click the unsubscribe button instead of shitposting

**Comment by Winne4r** [comment]
Although self-hosting is cool, the minimum system hardware requirements for Sentry are as follows:

- 4 CPU Cores
- 16 GB RAM

You may argue that it will not consume 4cores if you will send low amount of events... but clickhouse (main event db in sentry) still will steal a lot of cpu in background for some internal processing...

**Reply by blasdfaa** [reply]
Yes, that's a lot. Perhaps it will still be useful to someone

**Comment by orange1337** [comment]
+1

**Comment by Julhol-droid** [comment]
Does anybody know how to self host sentry in coolify? The Sentry Docs didn't help a much

**Reply by aybasaran** [reply]
https://github.com/getsentry/self-hosted 

this will guide i guess.

**Reply by Julhol-droid** [reply]
Well thanks, however this doesn't work because it seems that the docker compose file is too large for coolify to handle:
I get the following error message:

NOTE: BASE64_ENCODED_CONTENT_OF_DOCKER_COMPOSE_FILE is a placeholder, i added for github comment

**Here:**
`
docker exec lk0wooc8004oo40cswk0004o bash -c 'echo '${BASE64_ENCODED_CONTENT_OF_DOCKER_COMPOSE_FILE}'' | base64 -d | tee /artifacts/lk0wooc8004oo40cswk0004o/docker-compose.yml > /dev/null
`
Error: proc_open(): posix_spawn() failed: Argument list too long

**Comment by tylerdurden4285** [comment]
+1

**Comment by poeck** [comment]
+1

**Comment by elliotjarnit** [comment]
+1

**Comment by michalzarddev** [comment]
Please, this is a MUST have <3

**Comment by underthestars-zhy** [comment]
anyone figure out how to host it?

**Comment by debugHere** [comment]
+10

**Comment by pawelkrystkiewicz** [comment]
+1

**Comment by ItzAmirreza** [comment]
Looking forward to it.

**Comment by eduwass** [comment]
yes pls! 🙏

**Comment by compgeniuses** [comment]
whatever happened to this

**Comment by SegoviaDiego** [comment]
Not Sentry exactly, but a solid alternative while someone works on the sentry.io integration for Coolify:

The 🐐 @GauthierPLM has an open PR (#5386) with a working SigNoz setup. It’s expected to be merged in the next coolify release!

**Comment by 2u841r** [comment]
please @andrasbacsai

**Comment by iloveitaly** [comment]
Yeah sentry would be amazing!

**Comment by myagizmaktav** [comment]
+1

**Comment by colinmollenhour** [comment]


**Reply by eduwass** [reply]
whos gonna tell him

**Reply by colinmollenhour** [reply]
Ahh, is this a request for a template for running self-hosted Sentry? I think half the comments think it's for the template and the other half think it's some sort of feature request... 😄

**Comment by VettiMedia** [comment]
Seems like this guy got it running: https://medium.com/@dasezo/self-hosted-sentry-behind-coolify-a79dda1ceb6e

**Comment by erykkruk** [comment]
+1

**Comment by MikeeNachtigall** [comment]
+1

**Comment by doshibadev** [comment]
+1

**Comment by MichaelTendoSsemwanga** [comment]
+1

**Comment by tfbecker** [comment]
+1

**Comment by ravenelthomas** [comment]
+1

**Comment by EliaTolin** [comment]
+1
